ports/nrf/boards/SEEED_XIAO/deploy.md

```diff
@@ -0,0 +1,50 @@
+# Make and install a firmware package for SEEED nrf52840:
+
+1. Install adafruit-nrfutil with `apt-get install adafruit-nrfutil`
+   The official NRF nrfutil suite used in Makefile did not work for me and
+   failed consistently with Python errors. Maybe a wrong Python version,
+   but obviously it misses version checks.
+
+2. Create a firmware for the SEEED nrf52840 if needed, with the command
+
+   `make BOARD=SEEED_XIAO -j5`
+   
+   in the directory build-SEEED_XIAO-s140.
+
+3. Upload the firmware:
+
+   - generate in the build directory the nrfutil package with:
+
+     `nrfutil dfu genpkg --application firmware.hex --dev-type 82 firmware.zip`
+
+   - Set the board to dfu mode by pushing reset twice fast, until the
+     boards drive appears.
+
+   - Upload the firmware through USB serial with:
+
+     `nrfutil dfu serial -pkg firmware.zip --port /dev/ttyACM0 --baudrate 460800 -sb`
+
+In case you lost or have overwritten the XIAO bootloader, you can find it
+at https://github.com/Seeed-Studio/Adafruit_nRF52_Arduino.git in different
+formats. Using a JLINK adapter or interface, you can upload the hex version.
+The bootloader is as well available through the Arduino IDE.
+
+# Create a uploadable uf2 file.
```

Comment:
I'm not sure why this section is needed, since you added code in mpconfigboard.mk to automatically build the uf2.

Reply:
I'll remove it, but keep the note about reinstalling the bootloader.